packages/core/src/components/date-picker/utils/month-range.ts

```diff
@@ -0,0 +1,38 @@
+import defaultLocalization from "../date-localization";
+
+export function monthRange(selectedYear: number, minDate?: Date, maxDate?: Date): string[] {
```

Comment:
Dit moeten we denk ik of unit testen of e2e testen met cypress.

Reply:
Heb een cypress test gemaakt om te testen of de juiste opties worden getoond in de `select`.